pkg/NETStandard.Library.NETFramework/NETStandard.Library.NETFramework.pkgproj

```diff
@@ -143,5 +147,13 @@
         <TargetPath Condition="'%(LibFile.TargetPath)' == '' ">build/%(LibFile.TargetFramework)/lib%(LibFile.SubFolder)</TargetPath>
       </File>
     </ItemGroup>
+
+    <ItemGroup>
```

Comment:
talked offline and this is intentional to ensure we always include these in the src folder so that the common infra will correctly include them only in the symbol package.